projects/ion/src/lib/table/table.component.html

```diff
@@ -0,0 +1,111 @@
+<div class="ion-table">
+  <table data-testid="ion-table">
+    <tr class="border-header">
+      <th *ngIf="config.check">
+        <ion-checkbox
+          data-testid="table-check-all"
+          id="check-all"
+          name="check-all"
+          [state]="mainCheckBoxState"
+          (click)="checkState()"
+        >
+        </ion-checkbox>
+      </th>
+      <th
+        *ngFor="let column of config.columns"
+        [attr.data-testid]="'column-' + column.key"
+        [ngStyle]="{ width: column.width + '%' }"
+      >
+        <div class="header-style">
+          <span class="th-span">{{ column.label }}</span>
+          <button class="svg" (click)="sort(column)">
+            <svg
+              *ngIf="column.sort"
+              [attr.data-testid]="'sort-by-' + column.key"
+              width="8"
+              height="16"
+              viewBox="0 0 8 16"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M4.18015 0.103397C4.1005 -0.0344658 3.8995 -0.0344658 3.81985 0.103397L0.0276444 6.66693C-0.0513971 6.80373 0.0484804 6.97395 0.207794 6.97395H7.79221C7.95152 6.97395 8.0514 6.80373 7.97235 6.66693L4.18015 0.103397Z"
+                [attr.fill]="fillColor(column, true)"
+              />
+              <path
+                d="M3.81985 15.8966C3.8995 16.0345 4.1005 16.0345 4.18015 15.8966L7.97236 9.33307C8.0514 9.19627 7.95152 9.02605 7.79221 9.02605L0.207794 9.02605C0.0484809 9.02605 -0.0513966 9.19627 0.0276449 9.33307L3.81985 15.8966Z"
+                [attr.fill]="fillColor(column, false)"
+              />
+            </svg>
+          </button>
+        </div>
+      </th>
+      <th class="th-actions" *ngIf="config.actions && config.actions.length">
+        Ações
+      </th>
+    </tr>
+    <tr
+      *ngFor="let row of smartData; let index = index; let last = last"
+      [attr.data-testid]="'row-' + index"
+      [class.last-row]="last && !config.pagination"
+      [ngClass]="{
+        old: index % 2 === 0,
```

Comment:
if the table has no other cores to differentiate rows then it doesn't need the old class